src/components/button.css

```diff
@@ -0,0 +1,7 @@
+.c-button {
+  border-radius: 5px;
```

Comment:
is adding the trailing `;` something we can discuss? Because we don't use it in JS etc, it might make sense to not add it here